docs/dev/testnet/known-issues.md

```diff
@@ -72,3 +40,38 @@ zkSync does not support using CREATE/CREATE2 with raw bytecode. We highly recomm
 ## Hardhat's `console.log` does not work
 
 zkSync does not support the Nomic Foundation's `console.log` contract. Due to different address derivation rules, even when deployed, the `console.log` library will likely have a different address from the one on Ethereum.
+
+
+<!---
```

Comment:
Why did we add these as commented out code?

Reply:
It is now new info. It is issues that we already had, but are not relevant anymore. I didn't want to completely delete it, just in case.